// @progress/kendo-chat-vue-wrapper/dist/npm/KendoChat/Toolbar.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});
exports.default = {
    props: {
        toolbarAnimation: {
            type: Boolean,
            kComposite: 'toolbar.animation'
        },
        toolbarAnimationCollapseEffects: {
            type: String,
            kComposite: 'toolbar.animation.collapse.effects'
        },
        toolbarAnimationCollapseDuration: {
            type: Number,
            kComposite: 'toolbar.animation.collapse.duration'
        },
        toolbarAnimationExpandEffects: {
            type: String,
            kComposite: 'toolbar.animation.expand.effects'
        },
        toolbarAnimationExpandDuration: {
            type: Number,
            kComposite: 'toolbar.animation.expand.duration'
        },
        toolbarButtons: {
            type: Array,
            kComposite: 'toolbar.buttons'
        },
        toolbarScrollable: {
            type: Boolean,
            kComposite: 'toolbar.scrollable'
        },
        toolbarToggleable: {
            type: Boolean,
            kComposite: 'toolbar.toggleable'
        }
    }
};
